calibrations/mbd/prorun_mbdcal.sh

```
@@ -0,0 +1,277 @@
+#echo root.exe -q -b cal_mbd.C\(\"${inputs}\",${pass},${nevents},\"${dbtag}\"\)
+#root.exe -q -b cal_mbd.C\(\"${fname}\",${pass},${nevents},\"${dbtag}\"\)
+echo root.exe -q -b Fun4All_MbdCalReco.C\(\"${inputs}\",${pass},${nevents},\"${dbtag}\"\)
+root.exe -q -b Fun4All_MbdCalReco.C\(\"${inputs}\",${pass},${nevents},\"${dbtag}\"\)
+echo root.exe -q -b pro_cal_mbd.C\(${runno},${pass}\)
+root.exe -q -b pro_cal_mbd.C\(${runno},${pass}\)
+
+mv results/${runno}/mbd_qfit.root results/${runno}/mbd_qfit-${runno}.root
+
+echo "###############################################################################################################"
+echo "Running pass2.4 calibration"
+echo root.exe -q calib_t0mean.C\(\"results/${runno}/calmbdpass2.3_q_${runno}.root\"\)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)